cmd/logs.go

```diff
@@ -110,7 +134,9 @@ func newWorkersLogsCmd() *cobra.Command {
 astro deployment logs workers YOU_DEPLOYMENT_ID -s string-to-find
 `,
 		Args: cobra.ExactArgs(1),
-		RunE: workersRemoteLogs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return fetchRemoteLogs("worker", args)
```

Comment:
Same as previous comments, use constant for `worker`

Reply:
moved to a constant